src/components/UI/InputContainer/index.tsx

```diff
@@ -0,0 +1,111 @@
+import { styled } from '@/lib/stitches.config';
+
+export type Props = {
+  label?: string;
+  isError?: boolean;
+  errorMessage?: string;
+  children: React.ReactNode;
+};
+
+const InputContainer = (props: Props) => {
```

Comment:
다음과 같은 유틸타입으로 `children`과 prop을 분리할 수 있습니다.

```suggestion
const InputContainer = (props: PropsWithChildren<Props>) => {
```

Reply:
오 가독성이 훨씬 좋아보이는 것 같아요!
`PropsWithChildren`에서 children이 optional이길래 조금 고민이되어, 우선은 children을 가지고 있는 타입을 추가 정의해서 수정해두었습니다!  ☺️  2010970970c65374f0e1422abf6926ae18ed4a63